packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarItem.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { css, getModifier } from '@patternfly/react-styles';
+
+import {
+  DataToolbarBreakpointMod,
+  DataToolbarSpacer,
+  formatBreakpointMods,
+  formatItemSpacers
+} from './DataToolbarUtils';
+
+export enum DataToolbarItemMod {
+  separator = 'separator',
+  'bulk-select' = 'bulk-select',
+  'overflow-menu' = 'overflow-menu',
+  pagination = 'pagination',
+  'search-filter' = 'search-filter',
+}
+
+export interface DataToolbarItemProps extends React.HTMLProps<HTMLDivElement> {
+  /** Classes applied to root element of the Data toolbar item */
+  className?: string;
+  /** A type modifier which modifies spacing specifically depending on the type of item */
+  mod?: DataToolbarItemMod | 'separator' | 'bulk-select' | 'overflow-menu' | 'pagination' | 'search-filter';
```

Comment:
clear is only intended to be used alongside multiple chip groups in the expandable content so it will not be provided by the consumer - it will only ever be added (in the datatoolbar) programmatically.